### DISCUSSION TITLE
Why positive TF doesn't correlate with footprint?

### DISCUSSION THREAD

**Discussion by NoemieL** [GreenleafLab/ArchR#1637] [Questions / Documentation]
Hi,
I am comparing 2 groups of cells (with translocation (Blue)vs. without (Red)). I identified the motifs specific to the translocation group and looked for positive TFs. I identified TCF4 as the top motif in this group as well as being correlated with the gene expression (integrated scRNAseq). However, when I looked at the footprint of TCF4, I was expecting to see a significant difference between the two groups with the translocation group in blue having an higher accessibility around the motif but this is not the case. Could you please help me to understand these results? Thanks

# Motif enriched in the translocation group
![image](https://user-images.githubusercontent.com/26926109/191564078-1742a051-7c48-490e-a2d6-e8a1628db2f8.png)

# in blue are the TF identified in the translocation group and in red in the non translocation group. The correlation was performed using only the translocation-group cells.
![image](https://user-images.githubusercontent.com/26926109/191564207-21ab8891-9f41-4f64-8aef-069adf5ed5d1.png)

# TCF4 footprinting with different distance to motif centre
![image](https://user-images.githubusercontent.com/26926109/191564578-3891dc4e-58dc-430d-bbc7-254e329cda32.png)
![image](https://user-images.githubusercontent.com/26926109/191564784-a773f23b-76dd-4246-a9c1-02a1852bee0d.png)
![image](https://user-images.githubusercontent.com/26926109/191564632-3425356d-e8cd-4f53-a868-af1bdd66faf9.png)

**Comment by rcorces** [comment]
I dont like to comment on user-specific analyses. That being said, I would imagine that this is a problem of the peaks being used for the different analyses. I would assume that you did differential testing, found some peaks, and looked at motif enrichment within that peak subset. But then you did the footprinting off of all peaks?

**Reply by NoemieL** [reply]
Thanks Ryan, I really appreciate that you took time for this. 
I did the pairwise testing to identify the motifs with FDR<=0.01 and Log2FC>=1 following part 11 and 12 for the manual. I also looked at the motifs enrichment in marker peaks (FDR <= 0.01 & Log2FC >= 1) which give the same result since I have only two groups. In the footprint part of the manual I am not sure you mentioned to do the footprinting in a subset of peaks. How do you recommend to proceed?

**Reply by rcorces** [reply]
https://www.archrproject.com/reference/getFootprints.html
<html>
<body>


positions | A list or GRangesList of GRanges containing the positions to incorporate into the footprint. Each position should be stranded.
-- | --



</body>
</html>

**Reply by NoemieL** [reply]
Sorry, I already used the position parameter specifying TCF4. I did not understand what am I supposed to do. Should I filtered the ranges? If yes how am I supposed to do it (based on which criteria)? I tried using nTop to kept only the top ranges but it did not make any differences.
